pkg/watcherapi/statefulset.go

```diff
@@ -58,6 +60,12 @@ func StatefulSet(
 	readinessProbe.HTTPGet = livenessProbe.HTTPGet
 	startupProbe.HTTPGet = livenessProbe.HTTPGet
 
+	if instance.Spec.TLS.API.Enabled(service.EndpointPublic) {
+		livenessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+		readinessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+		startupProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+	}
```

Comment:
You are enabling https protocol for probes when TLS is enabled but you are actually not implementing it in the apache config so if someone enables TLS with this patch it will fail to start.

I understand you added this in the next patch in the series, and this is not a blocker but in case someone else test the patch in this state.

Reply:
that true, thanks @amoralej I forgot to add here the bit in the watcherapi controller where we configure TLS in the apache config, addint it now